composer.json

```diff
@@ -9,7 +9,9 @@
 		}
 	],
 	"require": {
-		"php": "^8.0"
+		"php": "^8.0",
+		"baraja-core/cms": "^4.0",
+		"baraja-core/doctrine-fulltext-search": "^3.0"
```

Comment:
Fulltextové vyhledávání potřebujeme proč?

Reply:
Chtěl jsem jen doctrine, nějak mi to ujelo.